src/Control/Monad/Free.purs

```diff
@@ -94,6 +95,10 @@ pureFC = liftFC <<< pure
 mapF :: forall f g a. (Functor f, Functor g) => Natural f g -> Free f a -> Free g a
 mapF t fa = either (\s -> Free <<< t $ mapF t <$> s) Pure (resume fa)
 
+-- | Use a natural transformation to interpret one `Free` monad as another.
+bindF :: forall f g a. (Functor f, Functor g) => Natural f (Free g) -> Free f a -> Free g a
```

Comment:
Just one small comment, the order of parameters does not match `bind`. Otherwise, :+1: